_includes/ee-linux-install-reuse.md

```diff
@@ -137,56 +137,58 @@ You only need to set up the repository once, after which you can install Docker
 
 {% elsif section == "install-using-yum-repo" %}
 
-There are currently two versions of Docker EE Engine available:
+For Docker Enterprise Edition 2.1, use Docker EE Engine 18.09. Installation instructions start below the note.
 
-* 18.03 - Use this version if you're only running Docker EE Engine.
-* 17.06 - Use this version if you're using Docker Enterprise Edition 2.0 (Docker
-Engine, UCP, and DTR).
+> ***NOTE:*** If you need to run Docker EE 2.0, please see the following instructions:
```

Comment:
oh, erm.. this only applies to 17.06. Docker 18.03 was a "standalone" release, so should not be used to run UCP

(I know it's confusing; FWIW; 18.03 was released to bring back feature-parity with Docker CE for customers that only needed the "engine"; now that CE and EE are released in tandem again, 18.03 will become less important)

Reply:
@thaJeztah So remove line 140? It would be really helpful during the edit process if you could explicitly state what you need.